scripts/requirements.txt

```diff
@@ -0,0 +1 @@
+requests==2.25.1
```

Comment:
we are doing only simple GET and POST requests, let's use the standard library for that. then, we only need a Python 3 environment without any other dependencies to run this.